<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishsListForm</class>
 <widget class="QWidget" name="DishsListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Все виды блюд</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> 
 QWidget{
color: #333;
border: 2px solid #888888;
font: 16pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}
 
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>День:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dayLabel">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="trapezaLabel">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>Название трапезы:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="dishsListView">
     <property name="font">
      <font>
       <family>pt Arial</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QListView {
color: #333;
border: 1px solid #888888;
font: 36px pt &quot;Arial&quot;;
border-radius: 1px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton {
color: #333;
border: 2px solid #888888;
font: 30pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="toMainButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton {
color: #333;
border: 2px solid #888888;
font: 30pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>Домой</string>
     </property>
     <property name="icon">
      <iconset resource="images_resources.qrc">
       <normaloff>:/images/img/kfm_home_6893.png</normaloff>:/images/img/kfm_home_6893.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images_resources.qrc"/>
 </resources>
 <connections/>
</ui>
